lib/dhcp_wire.ml

```diff
@@ -651,10 +651,10 @@ let options_of_buf buf buf_len =
             (get_ip_tuple_list ())
       in
       let get_string () =  if len < 1 then invalid_arg bad_len else
-          Cstruct.to_string body ~off:0 ~len
+          Cstruct.to_string body
       in
       let get_client_id () =  if len < 2 then invalid_arg bad_len else
-          let s = Cstruct.to_string body ~off:1 ~len:(len - 1) in
+          let s = Cstruct.to_string ~off:1 body in
```

Comment:
same as the other comment

Reply:
done in [732a8c3](https://github.com/mirage/charrua/pull/123/commits/732a8c3063bcea838bf0b29b5d2597be9c1a5ace)